clients/ui/frontend/src/app/utils/useDeepCompareMemoize.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
```

Comment:
This hook is unused and also we have this in shared library. We can import it from there.